python/xorbits/_mars/dataframe/tests/test_arrays.py

```diff
@@ -469,9 +469,9 @@ def test_to_pandas():
     assert df2.memory_usage(deep=True).sum() < df.memory_usage(deep=True).sum()
 
     # test df method
-    df4 = df2.groupby("b").sum()
```

Comment:
Better have `numeric_only=False` covered by some test cases. 

I think we can create an issue for this for now and fix it later. What do you think?